app/wasm/test/custom_msg_test.go

```diff
@@ -544,3 +543,60 @@ func executeCustom(t *testing.T, ctx sdk.Context, osmosis *app.OsmosisApp, contr
 	_, err = contractKeeper.Execute(ctx, contract, sender, reflectBz, coins)
 	return err
 }
+
+//func TestLockTokensMsg(t *testing.T) {
```

Comment:
Is there a reason this is commented out?

Reply:
Basically cause I haven't finished writing the tests. I'm having some issues with the wasm bindings used as testdata (https://github.com/osmosis-labs/osmosis/pull/1445#issuecomment-1135695165). I should have some free time to look into this this week.

Reply:
Gotcha. Should be move this into Draft mode until it's all good-to-go?

Reply:
Fixed the tests. Might still be good to write some more, but closing this conversation for now